standards/peer-review.md

```diff
@@ -1,8 +1,33 @@
+## Pull Request Workflow
+
+- All Pull Requests to NYPL repositories MUST be reviewed prior to merging (caveat below).
+  - PRs MUST NOT be merged if a reviewed requests changes
+  - Teams MAY agree to specific written policies as to conditions when reviews are not required. Such a policy MUST be approved by a Tech Lead or Engineering Manager.
+  
+- Engineers MUST mark at least one (1) developer as a reviewer and SHOULD request reviews from all relevant engineers
+  - Reviewers MAY come from any team within NYPL Digital
+  - Teams MAY create [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) files to establish a default group of reviewers
```

Comment:
It looks like we generally use user handles in our `CODEOWNERS`. Any experience/interest in tagging by team? Or are our write/admin teams usually larger than the set of handles we put in CODEOWNERS? Just seems a shame to have another thing to update for on/off-boarding, but given that git teams establish actual access, maybe that's fine.

Reply:
Usually the write/admin teams are larger than what we put in CODEOWNERS, but tbh I'm cool with you managing your subteams how you see fit and if you want to make another subteam called `lsp-codeowners` or something that seems like a good idea!